engine/runtime/src/test/java/org/enso/interpreter/test/SignatureTest.java

```diff
@@ -353,6 +354,109 @@ public void binaryWithVec() throws Exception {
     assertEquals("binary.Bin", ok3.getMetaObject().getMetaQualifiedName());
   }
 
+  @Test
+  public void partiallyAppliedConstructor() throws Exception {
+    final URI uri = new URI("memory://partial.enso");
+    final Source src = Source.newBuilder("enso", """
+    from Standard.Base import Integer
+
+    type V
+        Val a b c
+
+    create x:V = x.a + x.b + x.c
+
+    mix a =
+      partial = V.Val 1 a
+      create partial
+    """, uri.getHost())
+            .uri(uri)
+            .buildLiteral();
+
+    var module = ctx.eval(src);
+    var mix = module.invokeMember("eval_expression", "mix");
+
+    try {
+      var res = mix.execute(7);
+      fail("No result expected: " + res);
+    } catch (PolyglotException ex) {
+      assertContains("Type error", ex.getMessage());
+      assertContains("expected `x` to be V", ex.getMessage());
+      assertContains("got V.Val[partial", ex.getMessage());
+      assertContains("a=1", ex.getMessage());
+      assertContains("b=7", ex.getMessage());
+      assertContains("c=_", ex.getMessage());
```

Comment:
can't we just make it a single line? This feels a bit old to split it into numerous "contains" asserts. Same comment in other cases.